### PR TITLE
[zuul] Move test-operator job to stages

### PR DESCRIPTION
**Pull request by lpiwowar**
The linked PR for the test-oprator role introduces the concept of
stages. This patch updates the job definition so that it is
consuming the newly defined interface. It is required because by
default only the tempest stage is enabled.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/2374

### CODE REVIEW

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
@eshulman2 FYI:

Failed because of:
```
  Warning  InspectFailed   4m26s (x577 over 129m)  kubelet            Failed to apply default image tag "{{ _test_operator_tempest_registry }}/{{ _test_operator_tempest_namespace }}/{{ _test_operator_tempest_container }}:current-podified": couldn't parse image name "{{ _test_operator_tempest_registry }}/{{ _test_operator_tempest_namespace }}/{{ _test_operator_tempest_container }}:current-podified": invalid reference format
```
https://logserver.rdoproject.org/15/215/824b0bc5fcba970c32b8f5b2ffab3f368e2b0940/github-check/podified-multinode-edpm-deployment-crc-test-operator/d62d19e/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tempest-tests-tempest-full-workflow-step-0-dtq65/tempest-tests-tempest-full-workflow-step-0-dtq65-describe

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

File: .zuul.yaml [hunk 1]
```diff
@@ -35,75 +35,80 @@
...
+              tempest.api.compute.servers.test_device_tagging
+              tempest.api.network.admin.test_dhcp_agent_scheduler.DHCPAgentSchedulersTestJSON
+              tempest.scenario.test_minimum_basic.TestMinimumBasicScenario.test_minimum_basic_scenario
+              tempest.scenario.test_stamp_pattern
+              tempest.scenario.test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_live_migration
+              tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_mtu_sized_frames
+              test_create_server_invalid_bdm_in_2nd_dict
+              tempest.api.identity.admin.v3.test_credentials.CredentialsTestJSON
+              tempest.api.identity.admin.v3.test_tokens.TokensV3TestJSON.test_rescope_token
+              tempest.api.identity.admin.v3.test_users.UsersV3TestJSON.test_update_user_password
```
Comment (lpiwowar) [inline .zuul.yaml:61]: @eshulman2 this is a correct job definition that should work with the change in the test-operator role, right?

It seems like the include_list and exclude_list values did not get propagate correctly? -> https://logserver.rdoproject.org/15/215/824b0bc5fcba970c32b8f5b2ffab3f368e2b0940/github-check/podified-multinode-edpm-deployment-crc-test-operator/909305c/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/tempests.test.openstack.org/tempest-tests-tempest.yaml

Comment (eshulman2) [reply .zuul.yaml:61]: should be resolved now

**Comment by eshulman2** [comment]
recheck

**Comment by eshulman2** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
We do not need to merge this as the cifmw PR is backward compatible.

**Pull request closed without merging**